### ISSUE TITLE
Can no longer update the page with code

### ISSUE THREAD

**Issue by SYSU-lulc** [facebookresearch/Mephisto#502]
I have been able to modify the code and update the corresponding page normally. But suddenly no matter how to change the code, the page cannot be updated. Even if the code is restored to the code before the crash, it will not help. The command line appears to be running normally as before. The page opened by the port number is no longer updated. 

I tried to clean the browser cache. But it didn't help. In addition, the browser console also shows normal. Finally, I also tried to reconfigure the new virtual environment and re-run the experiment with the sample code. But the web pages that run out are fixed, even if my code is not written like this obviously.

In short, no matter what I do, the web pages opened on port 3000 on this machine are fixed. I don't know what went wrong, so I came to ask for help. Thank you for your attention.

**Comment by JackUrb** [comment]
Hi @SYSU-lulc - you may have the server still running from an old deploy. Could you check `lsof -I :3000`?

**Comment by SYSU-lulc** [comment]
It works!!! Thank you very much!!!
